swagger_to/swagger.py

```diff
@@ -276,6 +291,8 @@ def parse_path(adict: MutableMapping[str, Any]) -> Tuple[Path, List[str]]:
     errors = []  # type: List[str]
 
     for method_id, method_dict in adict.items():
+        if method_id.startswith('__'):
```

Comment:
I think you don't need this check here. The `items()` return only the items stored in a mapping (_i.e._ the dictionary) and do not iterate over all the properties of an object.